EjerciciosT15/src/paquetecuatro/LecturaArchivo.java

```diff
@@ -40,7 +41,18 @@ public static void leerRegistros() {
                 numeroEstablecimientosEntero = Integer.parseInt(numeroEstablecimientos);
                 
                 // agregar código aquí
-                
+                inicial = nombreCanton.charAt(0);
+                switch(inicial){ 
+                    case 'A':
+                    case 'E':
+                    case 'I':
+                    case 'O': 
+                    case 'U': 
+                        cadenaFinal = String.format("%s%s\n",
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.